src/psyclone/psyir/symbols/containersymbol.py

```diff
@@ -163,6 +169,27 @@ def wildcard_import(self, value):
                             f"'{type(value).__name__}'")
         self._has_wildcard_import = value
 
+    @property
+    def intrinsic(self):
+        '''
+        :returns: whether or not this module is an intrinsic module.
+        :rtype: bool
+        '''
+        return self._intrinsic
+
+    @intrinsic.setter
+    def intrinsic(self, value):
+        '''
+        Set whether or not this is an intrinsic module import.
```

Comment:
You can get rid of the property description of its the same as the first param.

Reply:
Done

Reply:
Actually i'm not sure I understand what you mean here? Is this docs related?